Committing branch of css fixes

andrasguseo opened the pull request:
Further css tweaks have been implemented to make the mini calendar widget look good out of the box.
Before-after screenshot:
https://cloudup.com/cZYeVz85QBd

Central:
https://central.tri.be/issues/107191

Commit:
Committing branch of css fixes

Commit:
Fixing indentation

nefeline commented:
Hey @andrasguseo , looks good to me! Let's merge this :beers: !

Commit:
Polishing

Polished the code a bit.
Added checks for theme and Pro.
Added fixes for Adv List widget.

Commit:
Fixes for Divi Builder

As per https://central.tri.be/issues/107191#note-11
and resetting some css styling done by Divi Builder

Commit:
Updated readme.txt / changelog

andrasguseo commented:
Now it should be really good. :)

Commit:
Datepicker z-index

Added fix for datepicker z-index when using shortcode month view
Modified checking for ECPro
Fixing whiespaces

cliffordp committed:
Removed indentations from empty lines

cliffordp committed:
Rename index.php to match plugin dir name

cliffordp committed:
Text tweaks

- add "divi" plugin tag
- update plugin description text
- consistent capitalization of "Mini Calendar widget"

cliffordp committed:
Text tweaks

- add "divi" plugin tag
- update plugin description text
- correct the release date of v1.2.0
- consistent capitalization of "Mini Calendar widget"

cliffordp committed:
PHP 5.6

cliffordp committed:
translations

cliffordp committed:
Final tweaks

cliffordp committed:
Convert all indents to tabs

cliffordp commented on tribe-ext-divi-theme-compatibility.php:
```diff
@@ -0,0 +1,282 @@
+
+		/**
+		 */
+		public function fix_et_sidebar_style() {
```
1. Does this only apply to PRO? What about TEC without PRO?
1. Please split this into 2 separate methods, one for theme, one for builder plugin. This is just best practice for single responsibility of each method, since no one would have both theme + builder plugin.
1. Add docblocks to both of these new methods

andrasguseo replied:
Thanks for your comments. I still have a long way to go. :) I will address your comments.
1. Yes, it applies to PRO only as the two widgets the styling addresses are available only in PRO.
2 & 3. Will do

cliffordp reviewed (changes requested):
Overall solid work. I made quite a few commits since it was easier than logging a bunch of little todos for you (and wasn't sure I could request all of them clearly).

Please review the commits I made, just for your reference, make sure you're working off the latest version of this branch, and then make the one last requested change.

Note that I did not QA this, just looked at the code, so please QA it before sending back to Code Review.

cliffordp committed:
clarity over brevity in comments

cliffordp commented:
@andrasguseo please also tag the past version numbers at https://github.com/mt-support/tribe-ext-divi-theme-compatibility/releases

andrasguseo commented:
@cliffordp 
> please also tag the past version numbers
When I follow the link I see an empty page. 
Could you walk me through this once sometime?

Commit:
Splitting method

Split up the method as per recommend
Renamed methods for better understanding
Added docblocks
Removed some whitespaces

cliffordp commented on tribe-ext-divi-theme-compatibility.php:
```diff
@@ -0,0 +1,272 @@
+			}
+
+			add_action( 'wp_head', array( $this, 'fix_divi_widget_styles' ) );
+			add_action( 'wp_head', array( $this, 'fix_db_widget_styles' ) );
```
Please move the conditionals here so the actions are only added when needed.

andrasguseo replied:
Yeah, I was thinking about that and stupidly I didn't ask.

cliffordp replied:
Not stupid at all.

cliffordp commented on tribe-ext-divi-theme-compatibility.php:
```diff
@@ -0,0 +1,272 @@
+		public function fix_divi_widget_styles() {
+
+			// Checking if Events Calendar PRO is active
+			if ( class_exists( 'Tribe__Events__Pro__Main', false ) ) {
```
instead of `class_exists`, please do like this: https://github.com/mt-support/tribe-ext-pdf-tickets/blob/1.1.0/tribe-ext-pdf-tickets.php#L104

Commit:
Moved conditionals

Moved conditionals so the actions are only loaded when needed.

cliffordp commented on tribe-ext-divi-theme-compatibility.php:
```diff
@@ -71,8 +71,22 @@ public function init() {
+				if ( is_plugin_active( 'divi-builder/divi-builder.php' ) ) {
+					add_action( 'wp_head', array( $this, 'fix_db_widget_styles' ) );
+				}
+			} // if ( class_exists( 'Tribe__Events__Pro__Main', false ) )
```
fix last comment to match actual code or just simply to something like `// if Events Calendar PRO is active`

Commit:
Fixing a closing if comment

cliffordp committed:
formatting fixes

cliffordp commented:
Quality work! 🥇
